### ISSUE TITLE
JCGS reviewer 1: (1) the argument presented is narrower?

### ISSUE THREAD

**Issue by TengMCing** [TengMCing/lineup_residual_diagnostics#11]
Reviewer 1: "I did not need any convincing that residual diagnostic plots are superior to tests but this paper presents a narrower argument to support this claim than many analysts would employ. For example, diagnostic plots may reveal features which are not directly connected to a linear model assumption and yet are interesting and important to identify. Of course, it is difficult to justify a journal article on the basis of such observations. In this respect, I am impressed that you have succeeded in demonstrating the superiority within the more formal framework of the lineup protocol. You have restricted attention to the residual-fitted plot and a particular selection of formal tests. I accept that this was necessary to make definite progress. Nevertheless, it would be good to add some discussion of the wider picture."

**Comment by TengMCing** [comment]
One of the benefits of using diagnostics plots over formal tests is they can provide additional information that is not directly related to model assumption. Perhaps we can add one or two paragraphs talking about this.

**Comment by dicook** [comment]
Thank for this feedback. We are happy to see that, like many people, you don't need convincing and that you are glad to see that we have succeeded in demonstrating the superiority of residual diagnostic plots. 

We have added a broader discussion on the use of residual plots in Section XXX, beyond those tested in this work. (Need to include: (1) finding interesting feature that was not intended to test for; (2) other residual plots - need to make a comprehensive list of common residual plots.)
